rest-api/README.md

```diff
@@ -0,0 +1,65 @@
+# Bareos REST API using fastAPI and python-bareos.
+
+Experimental and subject to enhancements and changes.
+
+This is an experimental backend for development purposes.
+
+It provides a REST API using fastapi and python-bareos to connect to
+a Bareos director.
+
+Connexion handles the HTTP requests defined by using OpenAPI
```

Comment:
This is no longer the case and relates to a previous approach.

Reply:
removed connexion remark